src/components/AuthErrorMessage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Message } from 'semantic-ui-react';
+
+const AuthErrorMessage = ({ error, hidden }) => {
+  const list = error
+    ? Object.keys(error).map(err => (
+      <p>
+        {err}
+:
+        {' '}
+        {error[err]}
+      </p>
+    ))
+    : null;
+  return (
+    <Message
+      hidden={hidden}
+      error
+      header="Oh! It seems something went wrong"
+      list={list}
+    />
+  );
+};
+
+AuthErrorMessage.propTypes = {
+  hidden: PropTypes.bool.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Please could you put this rule in eslintrc file so it would be used in similar cases

Reply:
This was just a temporary fixed I used for this particular line. it has been removed now.. thanks